C/c4CAPI.cc

```diff
@@ -413,6 +416,8 @@ C4Timestamp c4coll_nextDocExpiration(C4Collection *coll) noexcept {
 
 int64_t c4coll_purgeExpiredDocs(C4Collection *coll, C4Error * C4NULLABLE outError) noexcept {
     returnIfCollectionInvalid(coll, outError, 0);
+    C4CollectionSpec spec = c4coll_getSpec(coll);
+    Log("API purgeExpired in collection: %.*s.%.*s", SPLAT(spec.scope), SPLAT(spec.name));
```

Comment:
Similar comment as above. I think we don't need "API" there. Probably "Purge expired docs in collection : scope.collection" or "Purge expired docs in collection 'scope.collection'".

Reply:
okay